README.md

```diff
@@ -75,7 +89,9 @@ Like [ESLint](http://eslint.org/docs/user-guide/configuring#configuring-rules),
 }
 ```
 
-Some rules require options. There are no default values, so each rule that requires options must be explicitly configured:
+Some of the rules expect options. Each of these must be explicitly configured as there are no default values.
```

Comment:
I'd bold or italicize "there are no default values", because that's pretty important.